### PR TITLE
common/environment/check: enable offscreen test for qt

### PR DESCRIPTION
**Pull request by Chocimier**

### CODE REVIEW

**Comment by ericonr** [comment]
I think this PR should also remove it from all templates.

**Review by Johnnynator — APPROVED** [review]
lgtm

File: common/environment/check/no_display.sh [hunk 1]
```diff
@@ -0,0 +1 @@
+export QT_QPA_PLATFORM=offscreen
```
Comment (sgn) [inline common/environment/check/no_display.sh:1]: I think it's better to `export QT_QPA_PLATFORM=${QT_QPA_PLATFORM:-offscreen}`, some template defined it to other value.

**Comment by ericonr** [comment]
FWIW, we could take the opportunity to export `CTEST_OUTPUT_ON_FAILURE=TRUE` as well for the cmake build style (helps when checking CI results).

And IMO the package fixes can be a single commit for all packages, since it's always the same change. It helps when reviewing history.

**Comment by Chocimier** [comment]
All good points, done.

**Pull request merged**